eva-lib/src/main/java/uk/ac/ebi/eva/lib/MongoConfiguration.java

```diff
@@ -29,12 +29,14 @@
 import org.springframework.data.mongodb.core.mapping.MongoMappingContext;
 
 import uk.ac.ebi.eva.commons.models.converters.data.DBObjectToVariantEntityConverter;
+import uk.ac.ebi.eva.lib.config.EvaProperty;
 
 import java.io.IOException;
 import java.util.ArrayList;
 import java.util.List;
 
 @Configuration
+@Import(EvaProperty.class)
```

Comment:
This is the MongoConfiguration class, did you mistakenly refer to DBAdaptorConnector?
If this import is not there (in MongoConfiguration) then the repository tests fail, complaining there are no beans of type EvaProperties

(i.e.
`Caused by: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'uk.ac.ebi.eva.lib.MongoConfiguration': Unsatisfied dependency expressed through field 'evaProperties'; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type 'uk.ac.ebi.eva.lib.configuration.EvaProperties' available: expected at least 1 bean which qualifies as autowire candidate. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}`
)